python/manualPredictor.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+# Sets of labels corresponding to the 6 hyperplanes 
+hyperplanes = [('Driving', 'Stationary'), ('Driving', 'Using'), ('Driving','Walking'), ('Stationary' ,'Using'), ('Stationary', 'Walking'), ('Using','Walking')]    
+
+def manual_predict(datapoints, coefs, intercepts):
+    vectors_coefs_intercepts = zip(hyperplanes, coefs, intercepts) # Creating 3 tuples consisting of a label sets, vectors/coefs and intercepts
+    for point in datapoints:
```

Comment:
I get that you want to provide a list of datapoints, such that we can test if for multiple configs, but i think the predicter should only take one point as that is all a prediction function does: Take a point and classify it.
Than we can loop over the prediction function with different points. 
IMO a better approach.

Reply:
I agree with you that in general this is a better approach.